src/main/kotlin/org/rust/ide/refactoring/inlineValue/RsInlineValueProcessor.kt

```diff
@@ -33,9 +35,20 @@ class RsInlineValueProcessor(
     }
 
     override fun performRefactoring(usages: Array<out UsageInfo>) {
+        val factory = RsPsiFactory(project)
         usages.asIterable().forEach loop@{
             val reference = it.reference as? RsReference ?: return@loop
-            reference.element.replace(context.expr)
+            when (val element = reference.element) {
+                is RsStructLiteralField -> {
+                    if (element.colon == null) {
+                        element.addAfter(factory.createColon(), element.identifier)
```

Comment:
`element.identifier` can be null in the case of tuple struct fat initialization `S { 0: 0, 1: 1}`, how it would behave in this case? I suggest this:
```suggestion
                        element.addAfter(factory.createColon(), element.referenceNameElement)
```

Reply:
Oh, I had no idea you can initialize tuple structs like that. I added a test for it, but it works for the case that you have mentioned, because in that case the reference element is not `RsStructLiteralField`, but `RsPath`, so it works. And you cannot have shorthand initializer for tuple structs, so it should be fine.